.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,29 @@
+name: CodeQL
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ main ]
+  schedule:
+    - cron: '0 13 * * 3'
```

Comment:
Perhaps Tuesdays since our releases are scheduled for Wednesdays.

Reply:
good idea. will move to Monday.